tests/parse_metadata.py

```diff
@@ -30,6 +32,8 @@
     "read_stubtest_settings",
 ]
 
+OLDEST_SUPPORTED_PYTHON: Final = "3.7"
```

Comment:
TBH I haven't looked at the stub-uploader code yet. But I like the idea of defining the constant in pyproject.toml, I'll update the PR.

Reply:
For later, https://github.com/python/typeshed/blob/main/scripts/runtests.py#L31 could also use the specified stub's oldest python version instead of the common oldest supported overall.